### ISSUE TITLE
A11y: Table block Headline text has insufficient color contrast

### ISSUE THREAD

**Issue by ThomasKindermann** [plone/volto#2487]
The Headline text Color has insufficient color contrast (3.11:1) with the background of the Table Headline.
For better A11y it needs to have a color contrast of 4.5 or higher. 

**To Reproduce**
1. Add Table block to any page
2. Enter text in Headline Field
3. Open Axe-Dev and check site
4. Axe-Dev tool shows error

**Expected behavior**
Axe-Dev tool should show no color contrast violation for table headline
Color contrast should be above 4.5:1

**Screenshots**
![Screenshot 2021-06-02 at 15 54 18](https://user-images.githubusercontent.com/50659809/120499938-c0c1de00-c3c0-11eb-985c-74dfd38761fe.png)

![Screenshot 2021-06-02 at 16 39 52](https://user-images.githubusercontent.com/50659809/120500529-375edb80-c3c1-11eb-89af-86f51b667f21.png)

**Software:**
 - OS: OSX
 - Browser chrome
 - Volto Version "13.1.2"

**Comment by ThomasKindermann** [comment]
https://github.com/plone/volto/pull/2488 i changed the text color to black but now its harder to distinguish the headline from the other fields, any suggestions?

@sneridagh @tisto

**Comment by pnicolli** [comment]
Fixed by #6333
